hearth/lib/hearth/response.rb

```diff
@@ -17,6 +17,7 @@ def initialize(body: StringIO.new)
     # @param [Response] other
     # @return [Response]
     def replace(other)
+      other.body.rewind
```

Comment:
I don't think rewind should be done here for other. I think it should be the responsibility of the caller.

Reply:
Good catch, I didn't mean to check this change in.